roboteam_ai/src/conditions/IsInDefenseArea.cpp

```diff
@@ -36,11 +36,12 @@ bt::Node::Status IsInDefenseArea::update() {
 
     }
     ourDefenseArea = properties->getBool("ourDefenseArea");
+    outsideField = properties->getBool("outsideField");
```

Comment:
hasbool?

Reply:
getBool is fine here, if you want the behaviour to default to not being to go outside the field

Reply:
not needed